code/modules/projectiles/guns/energy/plasma.dm

```diff
@@ -102,4 +102,31 @@
 	overlays.Cut()
 	..()
 	if(cell)
-		overlays += image(icon, "cell_guild")
\ No newline at end of file
+		overlays += image(icon, "cell_guild")
+
+/obj/item/weapon/gun/energy/plasma/martyr // or should it be  Zealot
+	name = "NT PR \"Martyr\""
+	desc = "A \"NeoTheology\" weapon that uses advanced biomass conversion controllable blasts of energized matter. is a disposable side arm, good enough to save you and be recycled."
+	icon = 'icons/obj/guns/energy/martyr.dmi'
+	icon_state = "martyr"
+	suitable_cell = /obj/item/weapon/cell/martycharge
+	item_state = null	//so the human update icon uses the icon_state instead.
+	item_charge_meter = TRUE
+	w_class = ITEM_SIZE_SMALL
+	slot_flags = SLOT_BELT|SLOT_BACK|SLOT_POCKET
+	force = WEAPON_FORCE_PAINFUL
+	matter = list(MATERIAL_STEEL = 2, "biomatter" = 20)
+	disposable = TRUE
+	price_tag = 500
+	fire_sound = 'sound/weapons/Laser.ogg'
+	cell_type = /obj/item/weapon/cell/martycharge //can't recharge this one
+	sel_mode = 2
+	charge_cost = 1
+	recoil_buildup = 1 //pulse weapons have a bit more recoil
+	one_hand_penalty = 0
+	twohanded = FALSE
+
+	init_firemodes = list(
+		list(mode_name="Burn", projectile_type=/obj/item/projectile/plasma/stun, fire_sound='sound/weapons/Taser.ogg', fire_delay=2, charge_cost=null, icon="stun", projectile_color = "#0000FF"),
```

Comment:
Why charge cost is null for stun? Why stun mode is named burn?

Reply:
so it uses the default charge value. (i mostly try to keep things as the other stuff already in the code so instead of deleting the var from firemodes i just made it null. )

i'll delete it